src/index.spec.ts

```diff
@@ -139,45 +145,45 @@ describe('EppoClient E2E test', () => {
       });
     });
 
-    describe('UFC General Test Cases', () => {
-      it.each(readAssignmentTestData())(
-        'test variation assignment splits',
-        async ({ flag, variationType, defaultValue, subjects }: IAssignmentTestCase) => {
-          const client = getInstance();
-
-          let assignments: {
-            subject: SubjectTestCase;
-            assignment: string | boolean | number | object;
-          }[] = [];
-
-          const typeAssignmentFunctions = {
-            [VariationType.BOOLEAN]: client.getBoolAssignment.bind(client),
-            [VariationType.NUMERIC]: client.getNumericAssignment.bind(client),
-            [VariationType.INTEGER]: client.getIntegerAssignment.bind(client),
-            [VariationType.STRING]: client.getStringAssignment.bind(client),
-            [VariationType.JSON]: client.getJSONAssignment.bind(client),
-          };
-
-          const assignmentFn = typeAssignmentFunctions[variationType];
-          if (!assignmentFn) {
-            throw new Error(`Unknown variation type: ${variationType}`);
-          }
+    describe('Shared UFC General Test Cases', () => {
+      const testCases = testCasesByFileName<IAssignmentTestCase>(ASSIGNMENT_TEST_DATA_DIR);
```

Comment:
Makes for nice, unique test case names / output
<img width="710" alt="image" src="https://github.com/Eppo-exp/node-server-sdk/assets/417605/5d9d2201-8442-4b94-ba24-a725d4e9c2d4">